### PR TITLE
WDP240901-9

### PR DESCRIPTION
**Pull request by dave8git**
https://projects.kodilla.com/secure/RapidBoard.jspa?rapidView=239&projectKey=WDP240901&view=detail&selectedIssue=WDP240901-9

### CODE REVIEW

**Review by patrykbn — APPROVED** [review]
Seems good to me

**Comment by dave8git** [comment]
> Seems good to me

Thanks! :)

File: src/styles/settings.scss [hunk 1]
```diff
@@ -11,3 +10,19 @@ $primary: #d58e32;
...
+$transition-time: 250ms;
+$bg-transition-time: $transition-time;
+$text-transition-time: $transition-time;
+$motion-transition-time: $transition-time;
+$rotate-angle: 0deg;
+$scale-factor: 1.2;
+
+%interactive-hover {
+  transition: transform $motion-transition-time,
+    background-color $bg-transition-time, color $text-transition-time,
```
Comment (matwjablonski) [inline src/styles/settings.scss:25]: indentation and in the end of lines should be `;`

Comment (dave8git) [reply src/styles/settings.scss]: Identation fixed. 
Line ends fixed from , to ;

File: src/styles/settings.scss [hunk 2]
```diff
@@ -53,4 +52,7 @@ $scale-factor: 1.2;
     background-color $bg-transition-time, color $text-transition-time,
     border-color $transition-time, border-bottom $transition-time,
     visibility $transition-time;
-}
\ No newline at end of file
+}
+
+
```
Comment (matwjablonski) [inline src/styles/settings.scss:57]: why so many empty lines?

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: src/styles/settings.scss (hunk 2 above)
Comment (dave8git) [reply src/styles/settings.scss]: Fixed empty lines.

**Review by patrykbn — DISMISSED** [review]

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**